intermediate_source/autograd_saved_tensors_hooks_tutorial.py

```diff
@@ -19,13 +21,14 @@
 
 
 ######################################################################
-# 모델의 훈련은 일반적으로 모델을 인퍼런스하는 것보다 학습하는 과정에서 메모리를 더 많이 사용합니다.
-# 대략적으로 말하면 파이토치는 역전파를 호출하는데 필요한 계산 그래프를 저장해야하므로 추가 메모리 사용량이 있기 때문입니다.
-# 이 튜토리얼의 한가지 목표는 이러한 이해에 대한 미세 조정을 실행하는 것입니다.
+# 일반적으로 모델을 추론하는 것보다 학습하는 과정에서 메모리를 더 많이 사용합니다.
+# 대략적으로 말하면 파이토치는 역전파를 호출하는데 필요한 계산 그래프를 저장해야하므로
+# 추가 메모리 사용하기 때문입니다.
+# 이 튜토리얼의 목표 중 하나는 이런 내용을 이해와 미세 조정을 제공하는 것입니다.
 #
-# 사실, 이 그래프 자체로는 가끔은 그다지 많은 메모리를 사용하진 않습니다. 여타의 텐서들을 복제하지 않는 것 처럼요.
+# 사실, 가끔씩 계산 그래프 그 자체로는 복제 되지 않은 다른 tensor처럼 많은 메모리를 사용하지는 않습니다.
 # 그러나, 그래프는 *참조* 를 유지할 수 있습니다.
-# 그렇지 않으면 범위를 벗어난 텐서를 : **저장된 텐서** 라고 명명할 수 있습니다.
+# 그렇지 않으면 범위를 벗어난 tensor를 : **저장된 tensor** 라고 명명할 수 있습니다.
```

Comment:
원문의 한 문장을 쪼갠 것으로 보이는데요, 이전 문장과 함께 봐도 `saved tensor`에 대한 개념이 잘 전달되지 않는 것 같아서 변경이 필요할 것 같습니다.

`However, the graph can keep references to tensors that would otherwise have gone out of scope: those are referred to as saved tensors.`

대략 아래와 같은 정도는 어떨까요?

`하지만, 그래프는 범위에서 벗어난 tensor들에 대한 *참조(reference)* 는 유지할 수 있습니다 : 이러한 tensor들을 **저장된 tensor(saved tensor)** 라고 합니다.`